### ISSUE TITLE
Changing the value attribute on a form input field is not announced by NVDA

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1845]
**Reported by bgaraventa on 2011-10-17 05:31**
When JavaScript is used to dynamically change the value attribute on a form field. 
E.G.
document.getElementById('textField').value = 'NewValue';
The new value is not announced by NVDA.
An example of this issue is demonstrated at
http://whatsock.com/modules/auto_suggestion_search_field_module/demo.htm
Which uses the Up and Down arrow keys to dynamically change between selected values.

**Comment by ehollig** [comment]
Unfortunately, it appears that the link provided gives me a "406 Not Acceptable" error. Can anyone confirm if this issue appears in the latest Firefox with the latest NVDA? As @jcsteh said it works correctly in IE 8 from https://github.com/nvaccess/nvda/issues/1845#issuecomment-155293421, I would hope that the same results would occur in IE 11.

**Comment by ehollig** [comment]
Closing as worksforme as reported in https://github.com/nvaccess/nvda/issues/1845#issuecomment-155293421. Unfortunately, I am unable to test this myself, as stated in https://github.com/nvaccess/nvda/issues/1845#issuecomment-317496900, the link is unavailable. If anyone can still reproduce this issue and provide a working example, we can reopen this issue.
